books/RayTracingTheRestOfYourLife.html

```diff
@@ -1066,29 +1079,42 @@
 
     $$ f(d) = \sqrt{4\cdot\text{random_double}()} $$
 
-Note that this ranges from 0 to 2 as hoped, and if we check our work, we replace `random_double()`
-with $1/4$ to get 1, and also replace with $1/2$ to get $\sqrt{2}$, just as expected.
+Note that this ranges from 0 to 2 as we hoped, and if we check our work, we replace
+`random_double()` with $1/4$ to get 1, and also replace with $1/2$ to get $\sqrt{2}$, just as
+expected.
 </div>
 
+
+Importance Sampling
+--------------------
 <div class='together'>
-We can now sample our old integral:
+You should now have a decent understanding of how to take an analytical PDF and generate a function
+that produces random numbers with that distribution. We return to our original integral and try it
+with a few different PDFs to get a better understanding:
 
     $$ I = \int_{0}^{2} x^2 dx $$
 
-We need to account for the nonuniformity of the PDF of $x$. Failing to account for this
-nonuniformity will introduce bias in our scene, indeed this bias is the source of our inaccurately
-bright image, so if we account for nonuniformity, we will get accurate results. Where we sample too
-much we should down-weight. For our new nonuniform random number generator the PDF defines how much
-or how little we sample a specific portion. So the weighting function should be proportional to
-$1/\text{pdf}$. In fact it is exactly $1/\text{pdf}$:
+The last time that we tried to solve for the integral we used a Monte Carlo approach where we
+uniformly sampled between 0 and 2. We didn't know it at the time, but we were implicitly using a
```

Comment:
“… we used a Monte Carlo approach, uniformly sampling from the interval $[0,2]$.”